components/chainhook-cli/src/service/tests/helpers/mock_service.rs

```diff
@@ -1,33 +1,37 @@
-use crate::config::Config;
```

Comment:
This file is mostly just moving some test setup functions here from a different file, and adding the `TestSetupResult` struct to clean up the response a bit.